.github/workflows/auto-update.yml

```diff
@@ -0,0 +1,15 @@
+name: autoupdate
+on:
+  push:
+    branches:
+        - main
+jobs:
+  autoupdate:
+    name: autoupdate
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: docker://chinthakagodawita/autoupdate-action:v1
```

Comment:
Let's use a specific digest instead of a tag. So we can guarantee that the image is not replaced by something we didn't test.

Reply:
Done